Library/Formula/rbenv.rb

```diff
@@ -10,16 +10,37 @@ class Rbenv < Formula
   depends_on "ruby-build" => :recommended
 
   def install
-    inreplace "libexec/rbenv", "/usr/local", HOMEBREW_PREFIX
-    prefix.install Dir["*"]
+    inreplace "libexec/rbenv" do |s|
+      s.gsub!('"${BASH_SOURCE%/*}"/../libexec', libexec.to_s, false)
+      if HOMEBREW_PREFIX.to_s != "/usr/local"
+        s.gsub!(":/usr/local/etc/rbenv.d", ":#{HOMEBREW_PREFIX}/etc/rbenv.d\\0")
+      end
+    end
+
+    # Compile optional bash extension. Failure is not critical.
+    if File.exist? "src/configure"
```

Comment:
It wasn't present in 0.4.0 version, so this condition was for compatibility. Of course, it doesn't make sense now that the formula is bumped to 1.0.0 stable, but since the native extension is experimental and optional, maybe keep the conditional so that `--HEAD` installs keep working if this file gets changed in master?